server_src/player.js

```diff
@@ -911,7 +911,10 @@ class Player {
     if (!this.isBot && this.empTimer > 0) return; // emps don't stack. can't emp an already emp's ship. THIS IS A TEMPORARY MEASURE UNTIL THE EMP NONSENSE GETS FIXED
     if (this.ship >= 16 && this.ship<=20) t *= 1.5; // Emp works better on elites
     this.charge += -t*this.energy2; // Emp jams the ship. multiplying by energy2 ensures that regardless of energy tech, you remain jammed the same time
-    this.empTimer += t;
+    if (this.isBot) {
+      this.empTimer += t; // Adding a condition, checking that this doesn't cause the EMP bug.
```

Comment:
Should this be moved back out of the if statement, it looks like the empTimer wont update anymore for regular players.

Reply:
Actually, from what I see, the text message that comes later indicates how long they stay EMPd (actually jammed), and for some reason empTimer locks controls.

Reply:
So hopefully this way players just stay jammed while bots stay immobilized and control-locked

Reply:
Actually I have tested it now in the main server, it seems it has changed nothing player not bot-wise, it must be something related to the socket because it's the only other thing I have not touched when trying to fix it. 

Thta said, anything else I had except torpedo tweak range doesnt seem to be there

Reply:
This change hasnt been deployed to the main server yet

Reply:
Ah ok XD, sorry.